spec/controllers/groups_controller_spec.rb

```diff
@@ -21,8 +21,8 @@
 
   describe "GET #show" do
     it "assigns the requested user as @group" do
```

Comment:
I'm really sorry I didn't spot the word `user` here. I'm not sure if this is fixed in one of the other 2 PRs (which is confusing).